sequencer/src/bin/submit-transactions.rs

```diff
@@ -125,9 +125,11 @@ struct Options {
     #[clap(long, env = "ESPRESSO_SUBMIT_TRANSACTIONS_SUBMIT_URL")]
     submit_url: Option<Url>,
 
-    /// URL of the query service.
-    #[clap(env = "ESPRESSO_SEQUENCER_URL")]
-    url: Url,
+    /// list of query service URLs.
+    /// If `ESPRESSO_SUBMIT_TRANSACTIONS_SUBMIT_URL` is not provided,
+    /// transactions will be sent to a random node from this list.
+    #[clap(env = "ESPRESSO_SEQUENCER_URLS", value_delimiter = ',', num_args = 1..,)]
```

Comment:
This seems kind of difficult to work with. Like if someone sets these URLs and also sets `ESPRESSO_SUBMIT_TRANSACTIONS_SUBMIT_URL` the URLs will be ignored?

Could we just make `ESPRESSO_SUBMIT_TRANSACTIONS_SUBMIT_URL` accept a `Vec<_>` instead of `Opt<_>`?